### DISCUSSION TITLE
Block Protocol and Streamfields

### DISCUSSION THREAD

**Discussion by timonweb** [wagtail/wagtail#8001] [Ideas]
I wanted to draw contributors' attention to a new open initiative called the Block Protocol - https://blockprotocol.org/. It is an open standard for creating and using data-driven blocks and making them interoperable between applications. You can read more about the reasons for it on the initiative's author's blog: https://www.joelonsoftware.com/2022/01/27/making-the-web-better-with-blocks/.

I'm not sure if this will be useful or if it makes sense to make streamfields to comply with the protocol, but I wanted to let you know about this development.

**Comment by zerolab** [comment]
Thank you @timonweb. Converted this to a discussion (as it is better suited for that).
I recall seeing it briefly in the context of the [Wagtail page editor 2022](https://github.com/wagtail/wagtail/discussions/7739) (it probably was an offline discussion though)

**Comment by lb-** [comment]
This just popped up on my radar - the Block Protocol is a really interesting idea. They're about to update the schema version and launch a WordPress plugin.

https://blockprotocol.org/wordpress#availability
